cobalt/site/docs/development/setup-android.md

```diff
@@ -311,6 +311,22 @@ Cobalt-related output.
 As with the Linux build, use the `debug`, `devel`, or `qa` configs to trace
 Cobalt's callstacks.
 
+### Build Cobalt library locally
+
+**Partners should always use the Google prebuilt binaries from GitHub for certification
```

Comment:
Let's directly link the GitHub binaries for quick access as well:
- [Google prebuilt binaries from GitHub](https://github.com/youtube/cobalt/releases)

Reply:
Thanks for the suggestion. It is updated.